core/src/main/scala/org/http4s/HttpVersion.scala

```diff
@@ -33,6 +33,10 @@ final case class HttpVersion private[HttpVersion] (major: Int, minor: Int)
   override def render(writer: Writer): writer.type = writer << "HTTP/" << major << '.' << minor
   override def compare(that: HttpVersion): Int =
     (this.major, this.minor).compare((that.major, that.minor))
+
+  @deprecated("Does not range check parameters. Will be removed from public API in 1.0.", "0.22.6")
+  def copy(major: Int = major, minor: Int = minor): HttpVersion =
```

Comment:
removed without replacement?

Reply:
The traditional `copy` signature doesn't allow an error channel, and I don't think it has much of a use case.

Reply:
Yep definitely not :)